src/__test__/utils/validateInputs.test.js

```diff
@@ -88,7 +88,27 @@ describe('validateUnit unit test', () => {
     expect(invalidResult).toEqual([errorMessages.ALPHANUM_DASH_SPACE]);
   });
 
-  it('Correctly invalidates should not be the same with existing projects', () => {
+  it('Correctly invalidate names with invalid characters (alphanum, space)', () => {
+    const invalidName = 'Track-1!';
+    const checks = [rules.ALPHANUM_SPACE];
+
+    const { isValid: isInvalid, results: invalidResult } = validateInputs(invalidName, checks);
```

Comment:
I think that the tests will be much easier to read if this is changed to `const { isValid, results } = validateInputs(...)`

Reply:
Yes, you're right. Changed accordingly.